client/src/main.tsx

```diff
@@ -1,10 +1,7 @@
-import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App.tsx'
 import './index.css'
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
-  <React.StrictMode>
     <App />
-  </React.StrictMode>,
```

Comment:
Why are we no longer using strict mode?

Reply:
Strict mode is a debug setting that runs everything twice to help devs catch side-effect bugs. However, it was causing issues as our text would render double, and I wasn't using the debug feature much anyways. You can read more here:
https://react.dev/reference/react/StrictMode

Reply:
Doesn't this mean there's probably a bug though?

I believe in the `move fast and break things` philosophy rn though.